docs/user-guide/secrets/index.md

```diff
@@ -267,7 +318,7 @@ spec:
   restartPolicy: Never
 ```
 
-##### Consuming Secret Values from Environment Variables
+**Consuming Secret Values from Environment Variables**
```

Comment:
Use hashes (which note headings in Markdown) instead of surrounding in asterisks